incubator/binding-risingwave.spec/src/main/scripts/io/aklivity/zilla/specs/binding/risingwave/streams/effective/create.function.python/client.rpt

```diff
@@ -32,9 +32,9 @@ write zilla:data.ext ${pgsql:dataEx()
                               .query()
                                 .build()
                               .build()}
-write "CREATE FUNCTION gcd(int , int)\n"
+write "CREATE FUNCTION gcd(int, int)\n"
```

Comment:
Was this changed because it didn't work with the space?
If so, then is the grammar perhaps too strict?